src/azure-cli/azure/cli/command_modules/policyinsights/tests/latest/test_policyinsights_scenario.py

```diff
@@ -11,28 +11,38 @@
 
 class PolicyInsightsTests(ScenarioTest):
 
-    # Current recording was recorded against "Azure Governance Policy UX Test" (e78961ba-36fe-4739-9212-e3031b4c8db7)
+    # Current recording was recorded against "Azure Governance Perf 24" (3593b919-b078-4cc1-902f-201232a97ac0)
     @record_only()
-    @AllowLargeResponse()
+    @AllowLargeResponse(8192)
     def test_policy_insights(self):
+        self.kwargs.update({
+            'managementGroupId': 'azgovperftest',
+            'resourceGroup': 'PSTestRG1',
+            'subscriptionId': '3593b919-b078-4cc1-902f-201232a97ac0',
+            'keyVault': 'PSTestKV',
+            'subnet': 'PSTestVN',
+            'setDefinition': 'PSTestInitiative',
+            'definition': 'PSTestDINEDefinition',
+            'assignment': 'pstestdeployassignmentsub'
+        })
         top_clause = '--top 2'
         filter_clause = '--filter "isCompliant eq false"'
         apply_clause = '--apply "groupby((policyAssignmentId, resourceId), aggregate($count as numRecords))"'
         select_clause = '--select "policyAssignmentId, resourceId, numRecords"'
         order_by_clause = '--order-by "numRecords desc"'
-        from_clause = '--from "2021-07-01T00:00:00Z"'
-        to_clause = '--to "2021-07-03T01:30:00Z"'
+        from_clause = '--from "2022-04-01T00:00:00Z"'
+        to_clause = '--to "2022-04-03T01:30:00Z"'
         scopes = [
-            '-m "azgovtest5"',
+            '-m {managementGroupId}',
             '',
-            '-g "defaultresourcegroup-eus"',
-            '--resource "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/cheggpolicy/providers/microsoft.keyvault/vaults/cheggtmpkv"',
-            '--resource "cheggkv" --namespace "microsoft.keyvault" --resource-type "vaults" -g "cheggpolicy"',
-            '--resource "subnet2" --namespace "microsoft.network" --resource-type "subnets" --parent "virtualnetworks/cheggvnet" -g "cheggpolicy"',
-            '-s "1f3afdf9-d0c9-4c3d-847f-89da613e70a8"',
-            '-d "34c877ad-507e-4c82-993e-3452a6e0ad3c"',
-            '-a "4d31128e32d04a0098fd536e"',
-            '-a "f32eeddfc32345b585f9a70b" -g "cheggpolicy" '
+            '-g {resourceGroup}',
+            '--resource "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroup}/providers/microsoft.keyvault/vaults/{keyVault}"',
```

Comment:
Why do you want to replace the subscription id from `00000000-0000-0000-0000-000000000000` to `00000000-0000-0000-0000-000000000000`? Please don't replace it and try again

Reply:
this fixed it, thank you!